plugins/source/aws/resources/services/fsx/filesystems.go

```diff
@@ -0,0 +1,546 @@
+						Type:        schema.TypeBigInt,
+					},
+					{
+						Name:        "copy_tags_to_backups",
+						Description: "A Boolean value indicating whether tags on the file system should be copied to backups",
+						Type:        schema.TypeBool,
+					},
+					{
+						Name:        "copy_tags_to_volumes",
+						Description: "A Boolean value indicating whether tags for the volume should be copied to snapshots",
+						Type:        schema.TypeBool,
+					},
+					{
+						Name:        "daily_automatic_backup_start_time",
+						Description: "A recurring daily time, in the format HH:MM",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "deployment_type",
+						Description: "Specifies the file-system deployment type",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "disk_iops_configuration_iops",
+						Description: "The total number of SSD IOPS provisioned for the file system.",
+						Type:        schema.TypeBigInt,
+						Resolver:    schema.PathResolver("DiskIopsConfiguration.Iops"),
+					},
+					{
+						Name:        "disk_iops_configuration_mode",
+						Description: "Specifies whether the number of IOPS for the file system is using the system default (AUTOMATIC) or was provisioned by the customer (USER_PROVISIONED).",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DiskIopsConfiguration.Mode"),
+					},
+					{
+						Name:        "root_volume_id",
+						Description: "The ID of the root volume of the OpenZFS file system.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "throughput_capacity",
+						Description: "The throughput of an Amazon FSx file system, measured in megabytes per second (MBps)",
+						Type:        schema.TypeBigInt,
+					},
+					{
+						Name:        "weekly_maintenance_start_time",
+						Description: "A recurring weekly time, in the format D:HH:MM",
+						Type:        schema.TypeString,
+					},
+				},
+			},
+			{
+				Name:        "aws_fsx_filesystem_windows_configuration",
+				Description: "The configuration for this Microsoft Windows file system.",
+				Resolver:    schema.PathTableResolver("WindowsConfiguration"),
+				Columns: []schema.Column{
+					{
+						Name:        "filesystem_cq_id",
+						Description: "Unique CloudQuery ID of aws_fsx_filesystems table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "active_directory_id",
+						Description: "The ID for an existing Amazon Web Services Managed Microsoft Active Directory instance that the file system is joined to.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "aliases",
+						Description: "An array of one or more DNS aliases that are currently associated with the Amazon FSx file system",
+						Type:        schema.TypeJSON,
+					},
+					{
+						Name:        "audit_log_configuration_file_access_audit_log_level",
```

Comment:
What do you think about removing the `audit_log_configuration_` prefix?

Reply:
Mmm, good catch, thanks!